code/controllers/settings/combat_defines.dm

```diff
@@ -50,6 +50,7 @@
 	var/hmed_hit_damage_mult = 0.15
 	var/high_hit_damage_mult = 0.25
 	var/max_hit_damage_mult = 0.35
+	var/tacshottie_damage_mult = 0.25
```

Comment:
This is the same as `high_hit_damage_mult`. Also this kind of vars should be in the config files as well.

Reply:
No it's not, because we have a desynced config.txt:

Rohesie 2 days ago  Collaborator
This is -40%, not -25%
Again, document your changes.

  @Surrealaser
Surrealaser a day ago  Contributor
It's supposed to be -25%; I was referring to the ingame files which are inexplicably not synchronized with the external ones which are still the accurate/actual values.

  @Rohesie
Rohesie a day ago  Collaborator
Been like that since forever. Adjust accordingly and feel free to update the file.